src/test/java/com/datadoghq/datadog_lambda_java/XRayTraceContextTest.java

```diff
@@ -23,16 +23,48 @@ public void getAPMParentFromXray_happy_path() {
 
 
     @Test
-    public void getAPMParentFromXray_bad_characters(){
+    public void getAPMParentFromXray_bad_characters() {
         XRayTraceContext xrt = new XRayTraceContext();
         xrt.setParentId(";79014b90ce44db5e0;875");
         Assert.assertNull(xrt.getAPMParentID());
     }
 
     @Test
-    public void getAPMParentFromXray_too_short(){
+    public void getAPMParentFromXray_too_short() {
         XRayTraceContext xrt = new XRayTraceContext();
         xrt.setParentId("875");
         Assert.assertNull(xrt.getAPMParentID());
     }
+
+    @Test
+    public void getAPMTraceID() {
+        XRayTraceContext xrt = new XRayTraceContext();
```

Comment:
These look good. They are the same tests we have in the other languages right?

Reply:
Yeah, I ported these from Go directly